src/vqv.js

```diff
@@ -16,6 +16,14 @@
       #VQV!'
 */
 
-const vqv = (nome, idade) => {};
+const vqv = (nome, idade) => {
+  if (nome && idade) {
+    let string = `Oi, meu nome é ${nome}!
+Tenho ${idade} anos,
+trabalho na Trybe e mando muito em programação!
+#VQV!`;
+    return string
```

Comment:
Olá Vinicius! Pra funcionar o avaliador precisa consertar esse errinho de lint :)
```suggestion
    return string;
```